rules/retrieve.smk

```diff
@@ -56,6 +56,24 @@ if config["enable"]["retrieve"] and config["enable"].get("retrieve_databundle",
         script:
             "../scripts/retrieve_eurostat_data.py"
 
+    rule retrieve_jrc_idees:
+        output:
+            directory("data/bundle/jrc-idees-2021"),
+        log:
+            "logs/retrieve_jrc_idees.log",
+        retries: 2
+        script:
+            "../scripts/retrieve_jrc_idees.py"
+
+    rule retrieve_ammonia_demand:
+        output:
+            "data/bundle/myb1-2022-nitro-ert.xlsx",
```

Comment:
```suggestion
            "data/myb1-2022-nitro-ert.xlsx",
```
see above

Reply:
I will remove this rule as discussed below!